src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -1295,6 +1397,8 @@ class OMPLoopDirective(OMPRegionDirective):
 
     :param int collapse: optional number of nested loops to collapse into a \
         single iteration space to parallelise. Defaults to None.
+    :param kwargs: additional keyword arguments provided to the PSyIR node.
+    :type kwargs: unwrapped dict.
     '''
 
     def __init__(self, collapse=None, **kwargs):
```

Comment:
Since we're here, would you mind updating the user of `super` on the line below?

Reply:
Done